pom.xml

```diff
@@ -295,6 +296,7 @@
                 <versionOsgiRule implementation="org.eclipse.jetty.toolchain.enforcer.rules.RequireOsgiCompatibleVersionRule" />
                 <versionRedhatRule implementation="org.eclipse.jetty.toolchain.enforcer.rules.RequireRedhatCompatibleVersionRule" />
                 <versionDebianRule implementation="org.eclipse.jetty.toolchain.enforcer.rules.RequireDebianCompatibleVersionRule" />
+                <requireUpperBoundDeps/>
```

Comment:
This was accidentally deleted (by me).
This PR restores this 1 line, and the repercussions of it not being there.